devices/fields/dfsdm/f7.yaml

```diff
@@ -0,0 +1,316 @@
+
+DFSDM[0123]_ISR:
+  SCDF:
+    Clear: [0, No short-circuit detector event occurred on channel y]
+    Set:
+      [
+        1,
+        "The short-circuit detector counter reaches, on channel y, the value programmed in the DFSDM_CHyAWSCDR registers",
+      ]
+  CKABF:
+    Clear: [0, Clock signal on channel y is present.]
+    Set: [1, Clock signal on channel y is not present]
+  RCIP:
+    NotInProgress: [0, No request to convert the regular channel has been issued]
+    InProgress:
+      [1, The conversion of the regular channel is in progress or a request for a regular conversion is pending]
+  JCIP:
+    NotInProgress:
+      [0, No request to convert the injected channel group (neither by software nor by trigger) has been issued]
+    InProgress:
+      [
+        1,
+        "The conversion of the injected channel group is in progress or a request for a injected conversion is pending, due either to ‘1’ being written to JSWSTART or to a trigger detection",
+      ]
+  AWDF:
+    Clear: [0, No Analog watchdog event occurred]
+    Set:
+      [
+        1,
+        The analog watchdog block detected voltage which crosses the value programmed in the DFSDM_FLTxAWLTR or DFSDM_FLTxAWHTR registers,
+      ]
+  ROVRF:
+    Clear: [0, No regular conversion overrun has occurred]
+    Set:
+      [
+        1,
+        "A regular conversion overrun has occurred, which means that a regular conversion finished while REOCF was already ‘1’. RDATAR is not affected by overruns",
+      ]
+  JOVRF:
+    Clear: [0, No injected conversion overrun has occurred]
+    Set:
+      [
+        1,
+        "An injected conversion overrun has occurred, which means that an injected conversion finished while JEOCF was already ‘1’. JDATAR is not affected by overruns",
+      ]
+  REOCF:
+    Clear: [0, No regular conversion has completed]
+    Set: [1, A regular conversion has completed and its data may be read]
+  JEOCF:
+    Clear: [0, No injected conversion has completed]
+    Set: [1, An injected conversion has completed and its data may be read]
+
+DFSDM[0123]_ICR:
+  CLRSCDF: [0, 0xFF]
+  CLRCKABF: [0, 0xFF]
+  CLRROVRF:
+    _write:
+      Clear: [1, "Writing ‘1’ clears the ROVRF bit in the DFSDM_FLTxISR register"]
+  CLRJOVRF:
+    _write:
+      Clear: [1, "Writing ‘1’ clears the JOVRF bit in the DFSDM_FLTxISR register"]
+
+DFSDM[0123]_JCHGR:
+  JCHG: [0, 0xFF]
+
+DFSDM[0123]_FCR:
+  FORD:
+    FastSinc: [0, FastSinc filter type]
+    Sinc1: [1, Sinc1 filter type]
+    Sinc2: [2, Sinc2 filter type]
+    Sinc3: [3, Sinc3 filter type]
+    Sinc4: [4, Sinc4 filter type]
+    Sinc5: [5, Sinc5 filter type]
+  FOSR: [0, 1023]
+  IOSR: [0, 255]
+
+DFSDM[0123]_JDATAR:
+  JDATA: [0, 0xFFFFFF]
+  JDATACH: [0, 0x7]
+
+DFSDM[0123]_RDATAR:
+  RDATA: [0, 0xFFFFFF]
+  RDATACH: [0, 7]
+
+DFSDM[0123]_AWHTR:
+  AWHT: [0, 0xFFFFFF]
+  BKAWH: [0, 0xF]
+
+DFSDM[0123]_AWLTR:
+  AWLT: [0, 0xFFFFFF]
+  BKAWL: [0, 0xF]
+
+DFSDM[0123]_AWSR:
+  AWHTF: [0, 0xF]
+  AWLTF: [0, 0xF]
```

Comment:
This is 8bits from what I can tell from [rm0433](https://www.st.com/resource/en/reference_manual/rm0433-stm32h742-stm32h743753-and-stm32h750-value-line-advanced-armbased-32bit-mcus-stmicroelectronics.pdf) 30.8.11

Reply:
Now when I've look at this closer I think these fields need to split and collect in array.